### PR TITLE
Bundler 2.2.16

### PR DESCRIPTION
**Pull request by schneems**
- https://github.com/rubygems/rubygems/issues/4513
- https://github.com/rubyjs/libv8/issues/310#issuecomment-742481001
- https://github.com/rubygems/rubygems/pull/4497
- https://heroku.support/978905

### CODE REVIEW

**Review by edmorley — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Main (unreleased)
 
+## v226 (4/14/2021)
```
Comment (edmorley) [inline CHANGELOG.md:3]:
```suggestion
## v227 (4/14/2021)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+
+* Bundler 2.x is now 2.2.16 ()
```
Comment (edmorley) [inline CHANGELOG.md:5]:
```suggestion
* Bundler 2.x is now 2.2.16 (https://github.com/heroku/heroku-buildpack-ruby/pull/1150)
```

**Review by edmorley — APPROVED** [review]

**Comment by bf4** [comment]
lol, that's my support ticket :) thanks!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # Main (unreleased)
 
+## v227 (4/14/2021)
```
Comment (edmorley) [inline CHANGELOG.md:3]: The date here will need updating when the buildpack is published

**Pull request merged**